metaflow/runner/utils.py

```diff
@@ -107,12 +133,41 @@ def handle_timeout(
         )
         return content
     except (CalledProcessError, TimeoutError) as e:
-        stdout_log = open(command_obj.log_files["stdout"]).read()
-        stderr_log = open(command_obj.log_files["stderr"]).read()
-        command = " ".join(command_obj.command)
-        error_message = "Error executing: '%s':\n" % command
-        if stdout_log.strip():
-            error_message += "\nStdout:\n%s\n" % stdout_log
-        if stderr_log.strip():
-            error_message += "\nStderr:\n%s\n" % stderr_log
-        raise RuntimeError(error_message) from e
+        handle_process_error(command_obj)
+        raise RuntimeError from e
```

Comment:
we are raising `RuntimeError` again here, it was already done in `handle_process_error(command_obj)`

Please see the suggestion to return the error message and use it at once here..

Reply:
Fixed.